assets/src/block-editor/QueryBlockExtension/QueryBlockExtension.js

```diff
@@ -0,0 +1,126 @@
+import {RadioControl, PanelBody} from '@wordpress/components';
+import {InspectorControls} from '@wordpress/block-editor';
+import {useCallback, useEffect, useMemo, useState} from '@wordpress/element';
+
+const {addFilter} = wp.hooks;
+const {__} = wp.i18n;
+
+const targetBlocks = ['core/post-template'];
+
+const layoutTypes = [
+  {label: 'Grid', value: 'grid', columnCount: 3},
+  {label: 'Carousel', value: 'flex', columnCount: 6},
+  // {label: 'List', value: 'default', columnCount: 3}, // This might be enabled within Posts List
```

Comment:
Do we need to remove this commented line?

Reply:
We can remove it, but it is just for helping out the assigned dev whenever we work on Posts Lists